### ISSUE TITLE
[BUG]Managed Speed At Acceleration Altitude (video)

### ISSUE THREAD

**Issue by A320pilotIRL** [flybywiresim/a32nx#760]
**Mod Version**

MSFS 1.8.3.0. Bug was present before and after the mid September patch. 
Mod version 0.2.0 (latest stable release)

**Describe the bug**
 managed speed does not change as it should during a normal departure sequence. Described below

**To Reproduce**

1. Perform normal departure with airplane properly set up for takeoff
2. Observe TWO issues with the departure sequence, detailed below

**Expected behavior**

As soon as the aircraft lifts off the ground during takeoff, it should pitch to maintain a speed of V2+10. Flight Director bars should provide pitch guidance for V2+10, even though V2 will remain "bugged". When the airplane reaches the PERF programmed acceleration altitude, the airspeed bug should immediately change to the CLB speed, as the FMGS changes to the Climb Phase. The FD bars then direct the pilot to pitch for this speed, and the autothrust should immediately transition to THR CLB as soon as the thrust levers are moved to the climb detent. THR CLB should maintain thrust and not reduce it from 100% of the climb thrust, regardless of where the managed speed bug is. 

**Actual behavior**

Upon reaching acceleration altitude, the managed speed is targeting some weird climb speed (180kts if I recall in Conf 1+F) that seems to change with the flap position. The FD bars are following this and guiding a pitch to maintain this speed. This is completely in error and should never happen, even IF the plane is capable of pitching up to prevent flap over-speeds. The sim is also reducing thrust in THR CLB mode to maintain this speed (behaving as if it is in SPEED mode), which should NOT happen, but I'll put that in a different GitHub comment. 

**References**

Attached is an illustration video of what departure SHOULD look like. https://youtu.be/gLBlHOEQrIM

**Additional context**

Discord username (if different from GitHub): ragequitter 

TLDR: managed speed needs to immediately change to Climb phase speed at the acceleration altitude, Not maintaining some BS flap speed target until the flaps are moved.

**Comment by Venorcis** [comment]
See #565 as well; this report is probably more extensive though.

**Comment by PCGamesF1Tech** [comment]
I wrote a report via Zendesk about this strange autothrust behaviour and the FMA and EWD indications ("THR LVR" on FMA, changing thrust rating with lever position below CL notch, variable thrust with THR CLB,....)

**Comment by MisterChocker** [comment]
has been fixed by now
